OracleWebLogic/samples/12213-coherence-domain-in-image-wdt/Dockerfile

```diff
@@ -0,0 +1,148 @@
+#Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
+#
+#Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This Dockerfile extends the Oracle WebLogic image by creating a sample domain.
+#
+# Util scripts are copied into the image enabling users to plug NodeManager
+# automatically into the AdminServer running on another container.
+#
+# PREREQUISITES:
+# --------------
+# This sample requires that a Java JDK 1.8 or greater version must be installed on the machine
+# running the docker build command, and that the JAVA_HOME environment variable is set to the java home
+# location. The sample unzips the weblogic-deploy.zip into the image using the java jar command.
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+
+# Build the deployment archive file using the build-archive.sh script.
+#      $ ./build-archive.sh
+#
+# Run:
+#      $ sudo docker build \
+#            --build-arg CUSTOM_ADMIN_HOST=wlsadmin \
+#            --build-arg CUSTOM_ADMIN_PORT=7001 \
+#            --build-arg CUSTOM_ADMIN_NAME=7001 \
+#            --build-arg CUSTOM_MANAGED_SERVER_PORT=8001 \
+#            --build-arg CUSTOM_DOMAIN_NAME=base_domain \
+#            --build-arg CUSTOM_DEBUG_PORT=8453 \
+#            --build-arg WDT_MODEL=simple-topology.yaml \
+#            --build-arg WDT_ARCHIVE=archive.zip \
+#            --build-arg WDT_VARIABLE=properties/docker-build/domain.properties \
+#            --force-rm=true \
+#            --no-cache=true \
+#            -t 12213-domain-home-in-image-wdt .
+#
+# If the ADMIN_HOST, ADMIN_PORT, MS_PORT, DOMAIN_NAME are not provided, the variables 
+# are set to default values. (The values shown in the build statement). 
+#
+# You must insure that the build arguments align with the values in the model. 
+# The sample model replaces the attributes with tokens that are resolved from values in the
+# corresponding property file domain.properties. The container-scripts/setEnv.sh script
+# demonstrates parsing the variable file to build a string of --build-args that can
+# be passed on the docker build command.
+#
+# Pull base image
+# ---------------
+FROM container-registry.oracle.com/middleware/weblogic:12.2.1.3
```

Comment:
This should extend FROM oracle/weblogic:12.2.1.3-developer

Reply:
Ok, see my note is slack.

Reply:
Fixed